querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/DefaultNamingStrategy.java

```diff
@@ -33,7 +31,7 @@ public DefaultNamingStrategy() {
 
     @Override
     public String getClassName(String tableName) {
-        String className = tableName.substring(0, 1).toUpperCase(Locale.ENGLISH) +
+        String className = tableName.substring(0, 1).toUpperCase() +
```

Comment:
upercase tableName with system locale is not a good idea, java uses ENGLISH locale.
in example making className named "something" to "SOMETHİNG" in turkish doesn't work with java. 
query parameters should be loweCased and upperCased with system locale but field names, class names should use Locale.ENGLISH .